terraform/worker.tf

```diff
@@ -15,6 +15,12 @@ resource "cloudfoundry_app" "beis-roda-worker" {
   }
   service_binding { service_instance = cloudfoundry_service_instance.beis-roda-redis.id }
   service_binding { service_instance = cloudfoundry_service_instance.beis-roda-postgres.id }
+  service_binding {
+    service_instance = cloudfoundry_service_instance.beis-roda-s3-export-download-bucket.id
+    json_params      = <<EOF
+      {"permissions":"read-write"}
+    EOF
+  }
```

Comment:
@rjw1 Thanks bob, I'm now past those naming issues.

I'm getting an error now trying to set permissions on this `service_binding`:

```
Error: Unsupported argument

  on worker.tf line 20, in resource "cloudfoundry_app" "beis-roda-worker":
  20:     json_params      = <<EOF

An argument named "json_params" is not expected here.
```
Do you have any ideas how I can include an instruction here equivalent to the `-c '{"permissions":"read-write"}'` in:

```
cf bind-service beis-roda-staging-worker \
  s3-export-download-bucket-staging \
  -c '{"permissions":"read-write"}'
```

Reply:
https://registry.terraform.io/providers/cloudfoundry-community/cloudfoundry/latest/docs/resources/app#params